NHS.Ui/Web/NHSD_POC/static/NHSD_POC/js/trust_performance.js

```diff
@@ -22,7 +24,7 @@ var grapType =
 }
 
 function show_Hide_panel() {
-    var org_Code_value = document.getElementById("trust_list").value
+    var org_Code_value = document.getElementById("search").value
```

Comment:
Try to give some meaning full name to element 

like "trustsearch" or "trustsearchinput"